File: src/sdk-reference/go/1/collection/update-specifications/snippets/update-specifications.go
specifications := json.RawMessage(`{ "nyc-open-data": { "yellow-taxi": { "strict": false, "fields": { "license": { "mandatory": true, "type": "string" } } } } }`)
response, err := kuzzle.Collection.UpdateSpecifications("nyc-open-data", "yellow-taxi", specifications, nil)

if err != nil {
  log.Fatal(err)
} else {
  fmt.Println("Success")
}

File: src/sdk-reference/java/1/collection/update-specifications/snippets/update-specifications.java
try {
    String specifications = "{ \"strict\": false, \"fields\": { \"license\": { \"mandatory\": true, \"type\": \"string\" } } }";

    String response = kuzzle.getCollection().updateSpecifications("nyc-open-data", "yellow-taxi", specifications);

    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
